main.nf

```diff
@@ -493,24 +489,32 @@ if (!params.Q2imported){
 			script:
 			discard_untrimmed = params.retain_untrimmed ? '' : '--discard-untrimmed'
 			primers = single_end ? "--rc -g ${params.FW_primer}...${params.RV_primer}" : "-g ${params.FW_primer} -G ${params.RV_primer}"
-			int_out_files = single_end ? "-o firstcutadapt/${reads}" : "-o firstcutadapt/${reads[0]} -p firstcutadapt/${reads[1]}"
-			int_in_files = single_end ? "firstcutadapt/${reads}" : "firstcutadapt/${reads[0]} firstcutadapt/${reads[1]}"
+			in_2_files = single_end ? "second-trimming_${reads}" : "second-trimming_${reads[0]} second-trimming_${reads[1]}" 
+			out_1_files = single_end ? "-o second-trimming_${reads}" : "-o second-trimming_${reads[0]} -p second-trimming_${reads[1]}"
```

Comment:
This is a slightly confusing bit (it was before as well, at least to me), but shouldn't these out-files be "first_trimming"? Because I guess it's the out_files (without trimming in the names) that are the files that are used further along?

Reply:
Yes, thats quite confusing, I agree. The idea is that the input files to cutadapt are named in a way that describe its process, because the file names are picked up from the log files by MultiQC (and other parsers I guess). 
Meaning, I named the input file for the first round of cutadapt "first-trimming_" (`in_1_files`) and for the second cutadapt round "second-trimming_" (`in_2_files`, which are the output files of the first round: `out_1_files`).
So these names do not describe where they are coming from but where they are going into. I see that this is not intuitive.
If you'd like to see for yourself, just run `nextflow run d4straub/ampliseq -r dev -profile test_doubleprimers,singularity` and let it process until multiqc and look for the MultiQC report. If you have any idea how it might be solved/named better, I am open to ideas.

Reply:
I see what you mean and looking at the report it looks good.